DataFormats/Detectors/TRD/include/DataFormatsTRD/RawData.h

```diff
@@ -62,79 +62,86 @@ Word 7  |link 13 datasize lower |   link 14 errorflags   |              link 14
         -------------------------------------------------------------------------------------------------
 
 alternate candidate :
-        |31|30|29|28|27|26|25|24|23|22|21|20|19|18|17|16|15|14|13|12|11|10|09|08|07|06|05|04|03|02|01|00|
+        |00|01|02|03|04|05|06|07|08|09|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|
         -------------------------------------------------------------------------------------------------
-Word 0  | header version        | bunch crossing                    | stop bit  |  event type           |
+Word 0  | header version        | bunch crossing                    | stopbit| end point |  event type  |
         -------------------------------------------------------------------------------------------------
-Word 0  |                                          reserved                                             |
+Word 0  |                                          reserved 1                                           |
```

Comment:
Did we make a decision if we want to include a CRU ID in this header? This could go here, and would naturally also include the end point.

Reply:
What the hell was I thinking, yes will swap 0->31 and do 31->0 ..... 
As for FEEID and CRUID, that was still in the air as far as I know?

Reply:
Ok. We've got space and a header version in the right place. So let's got ahead.

Reply:
ok let me sort out the feeid and cruid question before finalising this PR.